courses/COMP-4400/Overview.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: overview
 title: COMP 4400 - Overview
-sidebar_label: COMP-4400
+sidebar_label: COMP-4400 - Overview

```

Comment:
Since these are nested under "COMP-4400" in the sidebar, should we drop the course code and just label them "Overview" and "Resources" respectively?

Reply:
idk I've been following what has been done before

Reply:
As long as its consistent I don't care. Looks good otherwise.